modules/analytics/src/views/full/index.tsx

```diff
@@ -48,6 +48,13 @@ interface State {
   pageTitle: string
   selectedChannel: string
   shownSection: string
+  disableAnalyticsFetching?: boolean
```

Comment:
What's that for ? I imagine this is for secret import preview ?

Reply:
Yeah, just so it doesn't trigger the fetching when the date changes